src/edu/csus/ecs/pc2/clics/API202306/EventFeedStreamer.java

```diff
@@ -864,31 +929,32 @@ public synchronized void sendJSON(String string) {
 
         string = replaceEventId (string, newId);
 
+        // Add notification to log before sending to clients
```

Comment:
It might be nice (i.e., to help future developers) if this comment include an explanation of WHY we are now putting the event in the log BEFORE sending it instead of after sending it.